docs/section5/Rule5-17.md

```diff
@@ -0,0 +1,48 @@
+
+# Envelope - Rule 5-17  
+
+**Rule ID:** 5-17  
+**Rule Description:** Opaque surfaces that are not regulated (not part of opaque building envelope) must be modeled the same in the baseline as in the proposed design.  
+**Appendix G Section:** Section G3.1-5 Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  - get_surface_conditioning_category()
+  - get_opaque_surface_type()
+  - match_data_element()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- For each building segment in the Proposed model: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal_block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`  
+
+      - For each surface in zone: `for surface_b in zone_b.surfaces:`  
+
+        - Check if surface is unregulated: `if ( scc_dictionary_b[surface_b.id] == UNREGULATED ):`  
+
+          - Get surface type: `surface_type_b = get_opaque_surface_type(surface_b)`
+
+          - Get matching surface from P_RMR: `surface_p = match_data_element(P_RMR, surfaces, surface_b.id)`  
+
+            **Rule Assertion:**  
+
+            - Case 1: If surface type is roof, floor or above-grade wall, and surface construction U-factor in B_RMR matches P_RMR: `if ( surface_type_b in ["ROOF", "FLOOR", "ABOVE-GRADE WALL"] ) AND ( surface_b.construction.u_factor == surface_p.construction.u_factor ): PASS`
+
```

Comment:
@xingcx this looks good. the only additional test we need for each case is the modeling method, which should be set to layer by layer input in both B and P RMR.

Reply:
I don't think App G requires that for unregulated surfaces, so I think this may be handled as part of general equivalency check that the same modelnig approach is used in P-RMR and U-RMR.

Reply:
@xingcx I agree, no changes required.